Lib/test/test_typing.py

```diff
@@ -3944,6 +3948,18 @@ class Cat(Animal):
             'voice': str,
         }
 
+    def test_is_typeddict(self):
+        assert is_typeddict(Point2D) is True
+        assert is_typeddict(Union[str, int]) is False
+        # classes, not instances
+        assert is_typeddict(Point2D()) is False
```

Comment:
@Fidget-Spinner, avoid naked asserts in the future, they produce bad error messages.

Reply:
Yeah I agree. Wait a minute, I don't remember this being part of the original commit 😕 . I think some other PR got caught in.

Reply:
Yeah, I fixed it.

Reply:
Alright thanks!